src/_support/Step/Acceptance/Administrator/Admin.php

```diff
@@ -106,21 +106,19 @@ public function __construct(Scenario $scenario)
 	}
 
 	/**
-	 * Method to confirm message appear
+	 * Method to confirm system message appear
 	 *
-	 * @param   string  $message  The message to be confirm
-	 *
-	 * @Then I should see the :message message
+	 * @param   string  $text     The text of message
+	 * @param   int     $timeout  Number of seconds to wait
 	 *
 	 * @since   __DEPLOY_VERSION__
 	 *
 	 * @return  void
 	 */
-	public function iShouldSeeTheMessage($message)
+	public function seeSystemMessage($text, $timeout = TIMEOUT)
```

Comment:
`seeSystemMessageContainer` method already exist in Joomla Browser, just see if you can use that one.

Reply:
Cant find seeSystemMessageContainer in https://github.com/joomla-projects/joomla-browser/blob/develop/src/JoomlaBrowser.php